docs/source/usage.rst

```diff
@@ -51,4 +51,51 @@ This then makes LiberTEM that is running on `remote-host` available on your loca
 The user interface
 ------------------
 
-TODO
+After starting the server, you can open the GUI in your browser. By default it will be at http://localhost:9000 .
+At the beginning, the GUI shows a promt to create a local cluster or connect to a running one. The number of workers is preset with a number 
+that will likely give optimal performance on the given machine.
+
+..  figure:: ./images/use/create.png 
+
+After connection to a cluster, LiberTEM shows a button to start browsing for available files. On a local cluster that's simply
+the local filesystem.
+
+..  figure:: ./images/use/browse.png
+
+This opens the file browser dialogue. On top it shows the current directory, below it lists all files and subdirectories in that directory. You select an entry by clicking once on it.
+You can move up one directory with the ".." entry on top of the list. The file browser is still very basic. Possible improvements are discussed in `Issue #83 <https://github.com/LiberTEM/LiberTEM/issues/83>`_. Contributions are highly appreciated!
+
+..  figure:: ./images/use/open.png
+
+After selecting a file, you set the type in the drop-down menu at the top of the dialogue above the file name. After that you set the appropriate parameters that depend on the file type. Clicking on "Load Dataset" will open the file with the selected parameters. The interface and internal logic to find good presets based on file type and available metadata, validate the inputs and display helpful error messages is still work in progress. Contributions are highly appreciated! TODO documentation of all supported file types and their parameters.
+
+* K2IS: The file to open is the ``.gtg`` file.
```

Comment:
You can actually just select any file that is part of the data set, any of the .bin files or the .gtg file, as long as they follow the default naming scheme.